docs/README.md

```diff
@@ -1,50 +1,30 @@
 # Nextflow Documentation 
 
-Nextflow documentation is written using [Sphinx](http://www.sphinx-doc.org/) which 
-uses the [reStructuredText](https://en.wikipedia.org/wiki/ReStructuredText) file format.
+Nextflow documentation is written using [Sphinx](http://www.sphinx-doc.org/), [MyST](https://myst-parser.readthedocs.io/en/latest/) which is an extended version of Markdown for Sphinx, and the [Read The Docs theme for Sphinx](https://github.com/readthedocs/sphinx_rtd_theme).
 
-A quick intro to reStructuredText is available at [this link](http://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html).
 
-To edit and contribute to the documentation you only need a text editor to change the
-appropriate `.rst` files in this directory.
+## Dependencies
 
-Once you have edited the documentation files verify the your changes are correctly applied
-using the command below to generate the HTML files:
+The build dependencies can be installed with `pip`:
 
+```bash
+pip install sphinx=3.5.4 myst-parser sphinx_rtd_theme
```

Comment:
@bentsherman , maybe best to pin the version for `myst-parser` and `sphinx_rtd_theme` the docs are known to work with.

Reply:
Done

Reply:
Thanks Ben - really looking forward to this getting merged 🤩